### PR TITLE
Registration invitation status

### PR DESCRIPTION
**Pull request by santipalenque**
https://tipit.avaza.com/project/view#!tab=task-pane&groupby=MyTaskDueDate&view=vertical&task=3423319&stafilter=NotComplete&fileview=grid
https://tipit.avaza.com/project/view#!tab=task-pane&groupby=MyTaskDueDate&view=vertical&task=3423304&stafilter=NotComplete&fileview=grid

### CODE REVIEW

File: src/actions/attendee-actions.js [hunk 1]
```diff
@@ -74,60 +74,52 @@ export const setSelectedAll = (value) => (dispatch) => {
 const parseFilters = (filters, term = null) => {
     const filter = [];
 
-    if(filters.hasOwnProperty('statusFilter') && filters.statusFilter){
+    if (filters.tags?.length > 0){
+        filter.push(`tags==${filters.tags.map(t => t.tag).join('||')}`)
```
Comment (smarcet) [inline src/actions/attendee-actions.js:78]: could we use tag id instead ?

Comment (santipalenque) [reply src/actions/attendee-actions.js]: done

File: src/actions/attendee-actions.js [hunk 2]
```diff
@@ -74,60 +74,52 @@ export const setSelectedAll = (value) => (dispatch) => {
 const parseFilters = (filters, term = null) => {
     const filter = [];
 
-    if(filters.hasOwnProperty('statusFilter') && filters.statusFilter){
+    if (filters.tags?.length > 0){
+        filter.push(`tags==${filters.tags.map(t => t.id).join('||')}`)
```
Comment (smarcet) [inline src/actions/attendee-actions.js:78]: this should be tags_id not just tags

**Review by smarcet — CHANGES_REQUESTED** [review]
@santipalenque please review comments

File: src/components/forms/attendee-form/attendee-form.js [hunk 3]
```diff
@@ -253,6 +257,20 @@ class AttendeeForm extends React.Component {
                             error={this.hasErrors('company')}
                         />
                     </div>
+                    <div className="col-md-4">
+                        <label> {T.translate("edit_attendee.tags")}</label>
+                        <TagInput
+                          id="tags"
+                          clearable
+                          isMulti
+                          allowCreate
```
Comment (smarcet) [inline src/components/forms/attendee-form/attendee-form.js:266]: we dont need to allow to create here we need to remove **allowCreate**

File: package.json [hunk 4]
```diff
@@ -66,7 +66,7 @@
     "moment-duration-format": "^2.3.2",
     "moment-timezone": "^0.5.33",
     "node-sass": "^7.0.1",
-    "openstack-uicore-foundation": "4.1.57",
+    "openstack-uicore-foundation": "4.1.58",
```
Comment (smarcet) [inline package.json:69]: lets update uicore to latest

File: src/pages/attendees/summit-attendees-list-page.js [hunk 5]
```diff
@@ -669,6 +689,20 @@ class SummitAttendeeListPage extends React.Component {
               </div>
             </>
           }
+          {enabledFilters.includes('tags') &&
+            <div className="col-md-6" style={{minHeight: "61px", paddingTop: "8px"}}>
+              <TagInput
+                id="tags"
+                clearable
+                isMulti
+                allowCreate
```
Comment (smarcet) [inline src/pages/attendees/summit-attendees-list-page.js:698]: we dont need allow create here

**Review by smarcet — APPROVED** [review]
LGTM

**Pull request merged**